Deployment/Jenkins/master/Master-Ec2-Instance.groovy

```diff
@@ -0,0 +1,281 @@
+                  {
+                      "ParameterKey": "NoReboot",
+                      "ParameterValue": "${env.NoReboot}"
+                  },
+                  {
+                      "ParameterKey": "NoUpdates",
+                      "ParameterValue": "${env.NoUpdates}"
+                  },
+                  {
+                      "ParameterKey": "PipIndexFips",
+                      "ParameterValue": "${env.PipIndexFips}"
+                  },
+                  {
+                      "ParameterKey": "PipRpm",
+                      "ParameterValue": "${env.PipRpm}"
+                  },
+                  {
+                      "ParameterKey": "ProvisionUser",
+                      "ParameterValue": "${env.ProvisionUser}"
+                  },
+                  {
+                      "ParameterKey": "PyStache",
+                      "ParameterValue": "${env.PyStache}"
+                  },
+                  {
+                      "ParameterKey": "SecurityGroupIds",
+                      "ParameterValue": "${env.SecurityGroupIds}"
+                  },
+                  {
+                      "ParameterKey": "ServerHostname",
+                      "ParameterValue": "${env.ServerHostname}"
+                  },
+                  {
+                      "ParameterKey": "SubnetIds",
+                      "ParameterValue": "${env.SubnetIds}"
+                  },
+                  {
+                      "ParameterKey": "WatchmakerConfig",
+                      "ParameterValue": "${env.WatchmakerConfig}"
+                  },
+                  {
+                      "ParameterKey": "WatchmakerEnvironment",
+                      "ParameterValue": "${env.WatchmakerEnvironment}"
+                  }
+                ]
+              /
+            }
+        }
+    stage ('Prepare AWS Environment') {
+        options {
+            timeout(time: 1, unit: 'HOURS')
+        }
+        steps {
+            withCredentials(
+                [
+                    [$class: 'AmazonWebServicesCredentialsBinding', accessKeyVariable: 'AWS_ACCESS_KEY_ID', credentialsId: "${AwsCred}", secretKeyVariable: 'AWS_SECRET_ACCESS_KEY'],
+                    sshUserPrivateKey(credentialsId: "${GitCred}", keyFileVariable: 'SSH_KEY_FILE', passphraseVariable: 'SSH_KEY_PASS', usernameVariable: 'SSH_KEY_USER')
+                ]
+            ) {
+                sh '''#!/bin/bash
+                    echo "Attempting to delete any active ${CfnStackRoot}-Ec2Inst-${BUILD_NUMBER} stacks..."
+                    aws --region "${AwsRegion}" cloudformation delete-stack --stack-name "${CfnStackRoot}-Ec2Inst-${BUILD_NUMBER}"
+
+                    aws cloudformation wait stack-delete-complete --stack-name ${CfnStackRoot}-Ec2Inst-${BUILD_NUMBER} --region ${AwsRegion}
+                '''
+            }
+        }
+    }
+    stage ('Launch Jenkins Master EC2 Instance Stack') {
+        options {
+            timeout(time: 1, unit: 'HOURS')
```

Comment:
See prior option-redundancy note. If this _isn't_ a reduncancy, probably want to align the stage-timeout to the policy-timeout of the managed CFn stack-template.

Reply:
removing these